src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,41 @@
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  width: 800px;
+  padding: 10px;
+
+  @extend %start-day-sat;
+
+  @for $i from 1 through $month-length {
+    .calendar__day-#{$i} {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      height: 100px;
+      flex-basis: 100px;
+      flex-grow: 0;
+      background-color: #eee;
+      border: solid 1px black;
+    }
+
+    &__day-#{$i}::before {
+      content: '#{$i}';
+      font-size: 30px;
+    }
+  }
+
+  & > *:hover {
+    cursor: pointer;
+    background-color: #ffbfcb;
+    transition: 0.5s;
+    transform: translateY(-20px);
```

Comment:
The hover effect includes a `transform: translateY(-20px);` which might cause layout shifts when hovering over elements. Consider if this effect is desirable or if it might disrupt the user experience.